sensors/README.md

```diff
@@ -0,0 +1,90 @@
+# Sensor manager
+
+Module provides a basic interface for a variety of sensors.
+
+Assumptions:
+ - the client communicates with manager via messages to **/dev/sensors**,
+ - there is an external library **libsensors** containing sensor structures and message events - external API for the client
+ - sensors server takes sensors description as arguments list:  <sensor_name:bus:others>; arguments might be specific for the sensor driver,
```

Comment:
A bus icon has been appeared here ;d
Please escape it.
![Screenshot from 2022-06-09 14-03-50](https://user-images.githubusercontent.com/77304431/172843088-57ec5fd0-4b55-452b-8d24-243f15fcd31b.png)